exporter/observiqexporter/factory.go

```diff
@@ -60,6 +60,7 @@ func createLogsExporter(
 	params component.ExporterCreateSettings,
 	config config.Exporter,
 ) (exporter component.LogsExporter, err error) {
+	params.Logger.Warn("The observiq exporter is deprecated and will be removed in future versions.")
```

Comment:
Do you have a timeframe for when it will be removed?  If so would be good to add that here and in the README

Reply:
I think the standard 2 releases makes sense, so I've slated it for removal in v0.56.0.